client/wfprev-war/src/main/angular/src/app/components/shared-layout/app-header/app-header.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { ResourcesRoutes } from 'src/app/utils';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-app-header',
+  standalone: true,
+  imports: [
+    CommonModule, // Replace BrowserModule with CommonModule
+    MatMenuModule,
+    MatButtonModule,
+    MatIconModule,
+  ],
+  templateUrl: './app-header.component.html',
+  styleUrls: ['./app-header.component.scss']
+})
+export class AppHeaderComponent {
+
+  constructor(
+    protected router: Router,
+  ) {
+  }
+
+
+  environment:string = 'DEV'
+  title:string = 'PREVENTION'
+  currentUser: string = 'User_1'
+
+  onLogoutClick(){
+    //clear token will be implemented after authorization piece done
+    // this.tokenService.clearLocalStorageToken();
```

Comment:
Are we removing this intentionally?  Can we just delete vs. comment out?

Reply:
We will need this function once we have the authentication piece done to log out the user, so I think we better leave this in place.